### DISCUSSION TITLE
Support the deployment of mmagic algorithm CircleGAN on ONNXRuntime backend 支持图像转换模型CycleGAN在ONNXRuntime侧的部署

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#587] [MMDeploy]
# Support the deployment of mmagic algorithm CircleGAN on ONNXRuntime backend 
 支持图像转换模型CycleGAN在ONNXRuntime侧的部署 

Support the deployment of CircleGAN algorithm from mmagic on ONNXRuntime.
You can refer to the following resources:
1. [CircleGAN](https://github.com/open-mmlab/mmagic/tree/main/configs/cyclegan)
2. [MMagic Deployment]([https://mmdeploy.readthedocs.io/en/latest/04-supported-codebases/mmagic.html](https://mmdeploy.readthedocs.io/en/latest/04-supported-codebases/mmagic.html) )
3. [Deployment Tutorials]([https://mmdeploy.readthedocs.io/zh_CN/latest/tutorial/01_introduction_to_model_deployment.html](https://mmdeploy.readthedocs.io/zh_CN/latest/tutorial/01_introduction_to_model_deployment.html) )

在mmdeploy支持 CircleGAN 算法在ONNXRuntime侧部署。
相关资源
1. [CircleGAN](https://github.com/open-mmlab/mmagic/tree/main/configs/cyclegan)
2. [MMagic部署]([https://mmdeploy.readthedocs.io/en/latest/04-supported-codebases/mmagic.html](https://mmdeploy.readthedocs.io/en/latest/04-supported-codebases/mmagic.html) )
3. [模型部署教程]([https://mmdeploy.readthedocs.io/zh_CN/latest/tutorial/01_introduction_to_model_deployment.html](https://mmdeploy.readthedocs.io/zh_CN/latest/tutorial/01_introduction_to_model_deployment.html) )

## Task Information

**Repository**：MMDeploy

**Difficulty**：Medium

**Points**：40

**Estimated Time Consumption**：21 days

## Task Status

**Claimant's Github ID**：None

**Claim Time**：2023年08月28日

**PR Link**：None

## Points to Note

To claim the task, please reply `Claim the mission` in the comment section. After claiming the task, please add our assistant bot called MeowMewo on WeChat at: `openmmlabwx`.

After completing the task, please submit a PR with the title starting with [MMSIG]. After submission, please reply with your PR link in the comments section of this Discussion. We will review your work as soon as possible.

在评论区回复 `Claim the mission` 领取任务，领取任务后请及时添加小助手喵喵微信: `openmmlabwx`

完成任务后提交相关链接（如是PR，标题请以[MMSIG]开头）回复到本Disucssion评论区，我们会第一时间审核您的贡献。

**Comment by zafirshi** [comment]
Mark for resources:

- [CircleGAN](https://github.com/open-mmlab/mmagic/tree/main/configs/cyclegan)
- [MMagic Deployment](https://mmdeploy.readthedocs.io/en/latest/04-supported-codebases/mmagic.html)
- [Deployment Tutorials](https://mmdeploy.readthedocs.io/zh_CN/latest/tutorial/01_introduction_to_model_deployment.html)
